doc/rst/platforms/cray.rst

```diff
@@ -46,12 +45,12 @@ installed, do the following:
      ./hello6-taskpar-dist -nl 4
 
 
-This should be all that is necessary to use Chapel on a Cray XC or HPE
-Cray EX system.  If the installation setup by your system administrator
+This should be all that is necessary to use Chapel on an HPE Cray EX or XC
+system.  If the installation setup by your system administrator
 deviates from the default settings, or you are interested in other
-configuration options, see `Using Chapel on a Cray System`_ below.  If
+configuration options, see `Using Chapel on an HPE Cray System`_ below.  If
 instead you wish to build Chapel from source, continue on to `Building
-Chapel for a Cray System from Source`_ just below.
+Chapel for an HPE Cray System from Source`_ just below.
 
 Note that the Chapel module for HPE Cray EX systems does not yet have
 the gasnet communication layer built into it.  For multilocale execution
```

Comment:
"does not yet" -> "does not". Best practice on the EX is to use ``ofi``.

Reply:
@jhh67 : I interpret the current phrasing (note that I didn't write/change this, at least in this PR) as meaning "it could and may in the future [for people who want to compare the two, e.g.]."  This is what we did on the XC as well—though ugni was preferred, we still bundled gasnet as a fallback / point of comparison.  I think the same would ultimately be good on EX as well, though we obviously haven't prioritized that effort at all.

I could also imagine saying "does not currently" to nod to the fact that GASNet supports a EX-oriented conduit as well, while also potentially making it sound less like we're behind where we want to be (we're not) or will be working on it anytime soon (we could, but it hasn't been discussed).

Reply:
I liked "does not currently" sufficiently more than "not yet have" to push that as a change.  I also capitalized GASNet in their preferred way.